docs/source/installation.rst

```diff
@@ -125,6 +127,18 @@ If everything is working properly, you should see the following window show up.
 
    ./_static/soccer.png
 
+Docker Setup
+------------
+Instead of using a virtual machine, you can also install our software using Docker. 
+The Docker image has our tech stack and all the dependencies pre-installed with 
+a desktop GUI. The Docker setup should work on any platform (Windows, Mac, ARM, AMD, etc.).
+
+Before you start, make sure you have Docker installed on your computer. The steps for doing
+so can be found at https://docs.docker.com/engine/install/.
+
+Once you have Docker installed, please follow the steps for installing and using our RoboCup 
+image at `DockerHub
+<https://hub.docker.com/r/robojackets/robocup-software>`_.
```

Comment:
Can you provide the `docker run` command that we need to run this? Are the various `-p` flags still necessary?

Reply:
`docker run -p 6080:6080 -p 5901:5901 -p 8081:8081 -d rishiso/robocup-software:latest`